src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs

```diff
@@ -7,6 +7,7 @@
 using System.Collections.Generic;
 using System.Security;
 using System.Runtime.InteropServices;
+using System.Diagnostics;
 using System.Diagnostics.Contracts;
```

Comment:
Okay, sounds good to me. How do we feel about removing the EndContractBlock calls? I thought those were only useful if we were using Code Contracts.